bucket/capnp.json

```diff
@@ -0,0 +1,25 @@
+{
+    "version": "1.0.2",
+    "description": "Capnp Proto is an insanely fast data interchange format and capability-based RPC system",
+    "url": "https://capnproto.org/capnproto-c++-win32-1.0.2.zip",
```

Comment:
Could you please move the `url` and `hash` to the correct place as per the order here https://github.com/ScoopInstaller/.github/blob/main/.github/CONTRIBUTING.md#for-scoop-buckets ?

Reply:
Done.

Reply:
Sorry I just meant putting `url` and `hash` after `license`, you don't need `architecture` if the app is 32bit only.